docs/pages/getting_started/access_data.mdx

```diff
@@ -0,0 +1,99 @@
+# Getting Started: Accessing Data
+The developers of the Pelican Platform believe data is the lifeblood of science. That's why Pelican provides a suite of tools that make it easy for researchers to federate their data, giving them the ability to share objects from a broad range of data repositories and allowing them to connect their data with distributed computational capacity, such as the OSPool. For more information on Pelican, visit the [About Pelican](../index.mdx) page.
+
+## Before Starting
+
+### Assumptions
+
+Before using the Pelican client to interact with objects, this guide makes several assumptions:
+
+- You are on a computer where you have access to a terminal. The Pelican client is a command line tool.
+- You've already installed the version of Pelican appropriate for your system, and Pelican is accessible via your path. To test this on Linux, you can run
+
+  ```bash
+  which pelican
+  ```
+  which should output a path to the executable. If there is no output to this command, refer to the [Pelican installation docs](../install.mdx) to acquire a working installation.
+
+### Useful Terminology
+
+#### Core Concepts
+To learn our terminology and the basics of Pelican, we recommend you read through our [Core Concepts](../index.mdx) page to learn how Pelican works and how it can be useful to you.
+
+#### Objects
+Pelican should be thought of as a tool that works with federated *objects* as opposed to *files*. The reason for this is that calling something a file carries with it the connotation of a filesystem. File systems have extra features such as owners, permissions, etc., which Pelican does not implement. In addition, a *file* gives the connotation that the file is mutable, i.e., its contents can change without requiring a new name. Objects in a Pelican federation, however, should be treated as **immutable**, especially in any case where objects are pulled through a cache (which will be the case for almost all files in the [Open Science Data Federation](https://osg-htc.org/services/osdf), or OSDF). This is because the underlying cache mechanism, powered by [XRootD](https://xrootd.slac.stanford.edu/), will deliver whatever object it already has access to; if an object's contents change at the origin, the cache will remain unaware and continue to deliver the old object. In the worst case, when the cache only has a partial object, it may attempt to combine its stale version with whatever exists at the origin. Use object names wisely!
+
+#### Namespace Prefixes
```

Comment:
I'm conflicted about this section, because I'm worried it's likely to confuse users, but I think a basic understanding of namespace prefixes versus object names is *sometimes* useful for debugging purposes. What do others think? @aowen-uwmad?

Reply:
Also, this section will be likely removed in #1396, I'm not too worried about leaving this as is.